clang/include/clang/Sema/Sema.h

```diff
@@ -13108,14 +13109,16 @@ class Sema final : public SemaBase {
               ? ExpressionEvaluationContext::ImmediateFunctionContext
               : ExpressionEvaluationContext::PotentiallyEvaluated);
       if (FD) {
+        auto &Current = S.currentEvaluationContext(),
+             Parent = S.parentEvaluationContext();
```

Comment:
```suggestion
        auto &Current = S.currentEvaluationContext();
        auto &Parent = S.parentEvaluationContext();
```
I think this is a bit clearer, because I don’t think you meant to copy the parent evaluation context here, right?